permissions/UDisks.permission

```diff
@@ -1,16 +1,18 @@
 # -*- mode: sh -*-
 
-# FIXME: what exactly requires udisks???
-
-# x-sailjail-translation-catalog =
-# x-sailjail-translation-key-description =
+# x-sailjail-translation-catalog = sailfish-catalog
+# x-sailjail-translation-key-description = permission-la-udisks2
 # x-sailjail-description = Use UDisks2
-# x-sailjail-translation-key-long-description =
-# x-sailjail-long-description =
+# x-sailjail-translation-key-long-description = permission-la-udisks2_description
+# x-sailjail-long-description = Access signals and properties on UDisks2, allows to get the devices, lock, mount and format to be called.
```

Comment:
I would not mention UDisks2 here. This is user visible text (when included directly by an application) and they don't have any idea what that means.

Reply:
Yeah, that might be better now with 94588c1489e297c32c6da6ee193a4119ed181cc6 ?